core/templates/dev/head/services/CsrfTokenService.ts

```diff
@@ -18,12 +18,8 @@
 
 // This needs to be imported first instead of using the global definition
 // because Angular doesn't support global definitions and every library used
-// needs to be imported explicitly. Also, default imports do not go do not go
-// down well with Karma and thus the import-as syntax.
```

Comment:
@YashJipkate -- just to confirm, was the old stuff here (the part that's now been deleted) incorrect? Would be good to get an understanding of where the discrepancy arose. /cc @kevinlee12

Reply:
I think it was not wrong but we missed the flag `esModuleInterop` in tsconfig which allows for default imports. After adding the flag, imports work fine.

Reply:
@seanlip I have inferred this myself since the default export stuff wasn't working (maybe because of some Webpack or TS config state). 
Thanks, @ankita240796 for giving a fix to this!